### DISCUSSION TITLE
the 'show results count' shows 0 the first time a user opens  the page

### DISCUSSION THREAD

**Discussion by Rhoufeo** [microsoft-search/pnp-modern-search#2445] [Q&A]
As indicated in the title the ‘Show results count’ is showing 0 results in the ‘PnP - Search Results’ web part when a user opens the first time the page although there are more than 16K documents in the relevant document library. The count is then growing over the time when the user is returning to the page after some time but for many users it’s somehow misleading since their interpretation of the 0 is that there is nothing for them to view on the page :( … the PnP version used is: 4.6.1 … 

Does anyone have any idea?
Thanks in advance!!

**Comment by wobba** [comment]
Can you look at the JSON data coming back on the query call? If the data shows 0 in the count, then it's an API issue we cannot resolve.

**Reply by FranckyC** [reply]
@Rhoufeo can you share your Search Results Web Part default configuration, especially the query template and query text? Thanks!

**Reply by Rhoufeo** [reply]
... thanks for your answers.
Please find attached the webpartproperties.json of the Search Results Web Part: [webpartproperties.txt](https://github.com/microsoft-search/pnp-modern-search/files/9797842/webpartproperties.txt)

Many thanks!

**Reply by GangadharMFord** [reply]
We have also faced the similar issue (not a first-time user but even though full control site admin did the search with few keywords) were few search keywords returning search results count 0 even though it has the search results. We are at 4.7 search result version using.
Can anyone suggest on this issue.

**Reply by wobba** [reply]
@Rhoufeo do you have * set as the default query when no search term is entered?

**Reply by wobba** [reply]
@GangadharMFord your issue seems related to the user, not entering the page.

**Reply by GangadharMFord** [reply]
hi @wobba, can you explain what is about "not entering the page" means please? 
I have also checked; we are not using default query as * into the search query and thank you for guidance.

**Reply by wobba** [reply]
@GangadharMFord The issue of @Rhoufeo is that when the the user navigate to a page (entering the page) first time. Which could be ACL related.
